psydac/api/ast/expr.py

```diff
@@ -17,8 +17,9 @@
 from pyccel.ast.core import _atomic
 
 from sympde.core                 import Constant
-from sympde.topology             import ScalarField, VectorField
-from sympde.topology             import IndexedVectorField
+from sympde.topology.space       import ScalarTestFunction
+from sympde.topology.space       import VectorTestFunction
+from sympde.topology.space       import IndexedTestTrial
 from sympde.topology.derivatives import _partial_derivatives
 from sympde.topology.derivatives import _logical_partial_derivatives
```

Comment:
According to Python PEP 8 we should only use names starting with an underscore for **private** variables. Here we are importing these lists from another module, so we should definitely change their names.

Since you are using these variables a lot in this PR, I think it's appropriate to change their names now.

Reply:
it's used only once in this PR, and those changes will need to be in Psydac and Sympde

Reply:
You are right, we should change `sympde` first.